crates/sui-types/src/gas.rs

```diff
@@ -454,7 +445,7 @@ impl<'a> SuiGasStatus<'a> {
     pub fn summary(&self) -> GasCostSummary {
         let remaining_gas = self.gas_status.remaining_gas();
         let storage_cost = self.storage_gas_units;
-        // TODO: handle underflow how?
+        // MUSTFIX: handle underflow how?
```

Comment:
not clear this should happen but good to have a mustffix to review rather than a todo